hist/hist.go

```diff
@@ -118,14 +118,15 @@ func (h *Histogram) String() string {
 		" 99.99th: %d\n" +
 		" Max:     %d\n" +
 		"Stats:\n" +
-		" Average: %f\n" +
+		" Average (%s): %f\n" +
 		" Total requests: %d\n" +
 		" Elapsed Time (sec): %.4f\n" +
```

Comment:
should we also do the same for this field?

Reply:
This field is the overall test elapsed time, and it is always in seconds. This is usually much larger than the single query time, so I think it's better not to use the same unit.